__tests__/routes/actions.test.js

```diff
@@ -139,15 +139,15 @@ describe('Actions Router', () => {
           res = await request(app).get('/actions/987654321');
         });
 
-        it('responds with status 404', () => {
-          const expected = 404;
+        it('responds with status 400', () => {
+          const expected = 400;
           const actual = res.status;
 
           expect(actual).toBe(expected);
-        });
+        }); //? this needs to be 404 i think?
```

Comment:
This assumption is correct, this should, in fact, return a 404, as a search that turns up no results should return a `404 - Resource not found` response. 😄

Reply:
Good shout, thank you, I fixed it!